style.css

```diff
@@ -130,12 +129,22 @@ input[type="submit"]:hover {
     color: white;
 }
 .info{
-    margin-left: 25%;
-    margin-right: 25%;
+    font-size: larger;
     width: 50%;
+    margin-left: 25%;    
+    margin-right: 25%; 
+    text-align: center;   
+}
+.info td{
+    height:50px;
+}
+.info1{
     font-size: larger;
-    text-align: center;
-    margin-top: 40px;
+    width: 50%;
+    margin-left: 25%;    
+    margin-right: 25%; 
+    text-align: center;   
+    margin-top: 140px;
 }
 .center{
     margin-top: 2%;
```

Comment:
nit-picky but please indent your files correctly. if everyone pushes misaligned files it's going to be a hot mess.

Reply:
what files are indented wrong?